### ISSUE TITLE
Recommend renderers extension when working with .ipynb files

### ISSUE THREAD

**Issue by SiddhanthUnnithan** [microsoft/vscode#132009]
## TL;DR
Recommend the output renderers extension to the user when they open an .ipynb file.

## Overview
The Jupyter team intends to separate notebook renderers into their own web-enabled extension. This extension aims to support more output-rendering scenarios when users are viewing/editing notebooks in VS Code web.

Users must install the extension from the marketplace if they wish to view the supported renderers. This is an extra step for users to take once they've navigated to the web client and requires context-switching from their notebook file, to the extension marketplace, and back to their notebook. Users must be aware of which extension to install to render the output, and search for said extension in the marketplace.

To avoid the cognitive load of having to know which extension to download and context-switching from their notebook to the marketplace, we're suggesting that users get recommended the renderer extension after opening an .ipynb file. VS Code already includes the logic to correctly render the notebook (i.e., cells and some output instead of raw JSON). The idea would be that the detection logic used to render the notebook would also serve as the logic to show a toast notification recommending the renderers extension for the user to install.

This concept of recommending extensions based on the .ipynb file being opened can be extended to include other Jupyter/Notebook-compatible extensions down the line (e.g., Jupyter, Pyodide).
